src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -1600,6 +1601,101 @@ std::set<std::vector<element::Type>> jit_subtract_emitter::get_supported_precisi
     return {{element::f32, element::f32}, {element::i32, element::i32}};
 }
 
+/// Elu ///
+jit_elu_emitter::jit_elu_emitter(ov::intel_cpu::riscv64::jit_generator *host,
+                                 ov::intel_cpu::riscv64::cpu_isa_t host_isa,
+                                 float alpha,
+                                 ov::element::Type exec_prc)
+        : jit_emitter(host, host_isa, exec_prc),
+          alpha(alpha) {
+    prepare_table();
+    exp_emitter = std::make_unique<jit_exp_emitter>(h, host_isa, exec_prc);
+}
+
+jit_elu_emitter::jit_elu_emitter(ov::intel_cpu::riscv64::jit_generator *host,
+                                 ov::intel_cpu::riscv64::cpu_isa_t host_isa,
+                                 const std::shared_ptr<ov::Node> &node,
+                                 ov::element::Type exec_prc)
+        : jit_emitter(host, host_isa, exec_prc) {
+    const auto elu = ov::as_type_ptr<ov::op::v0::Elu>(node);
+    if (elu == nullptr) {
+        OV_CPU_JIT_EMITTER_THROW("Can't cast to ov::op::v0::Elu");
+    }
+    alpha = static_cast<float>(elu->get_alpha());
+    prepare_table();
+    exp_emitter = std::make_unique<jit_exp_emitter>(h, host_isa, node, exec_prc);
+}
+
+size_t jit_elu_emitter::get_inputs_num() const {
+    return 1;
+}
+
+size_t jit_elu_emitter::aux_gprs_count() const {
+    return exp_emitter->aux_gprs_count();
```

Comment:
Since we use data section with constants (table), we [need one more GPR](https://github.com/openvinotoolkit/openvino/blob/5c7db84cdc44d7de778f91d8505859055d80d024/src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_emitter.hpp#L152) to store pointer to this table. Please fix this formula then